packages/actor-rdf-resolve-quad-pattern-hypermedia/lib/MediatedLinkedRdfSourcesAsyncRdfIterator.ts

```diff
@@ -121,7 +122,7 @@ export class MediatedLinkedRdfSourcesAsyncRdfIterator extends LinkedRdfSourcesAs
   }
 
   protected shouldStoreSourcesStates(): boolean {
-    return this.aggregatedStore === undefined;
+    return this.aggregatedStore === undefined && !this.context.get(KeysInitQuery.disableCaching);
```

Comment:
I don't think we need it here, since aggregate stores always only exist in the context of a single query exec.